File: Python/1000-1999/1101.py
# -*- coding: utf-8 -*-

def main():
    while True:
        x, y = map(int, input().split())
        soma = 0

        if x <= 0 or y <= 0:
            break

        temp = x
        temp2 = y

        x = max(temp, temp2)
        y = min(temp, temp2)

        for c in range(y, x+1):
            print('%i' %(c), end=' ')
            soma += c
        
        print('Sum=%i' %(soma))

main()

File: C/1000-1999/1101.c
#include <stdio.h>

int main (void){
  int m = 1, n = 1, soma;

  while (m >= 0 && n >= 0){
    soma = 0;
    scanf("%d %d", &m, &n);

    if (m <= 0 || n <= 0){
        break;
    } else if (m > n){
        for (int i = n; i <= m; i++){
            printf("%d ", i);
            soma += i;
        }
    } else{
        for (int i = m; i <= n; i++){
            printf("%d ", i);
            soma += i;
        }
    }

    printf("Sum=%d\n", soma);
  }

  return 0;
}
